models/comments-model.js

```diff
@@ -0,0 +1,47 @@
+const db = require("../db/connection");
+const { selectArticles } = require("../models/articles-model");
+
+exports.selectComments = (articleId) => {
+  if (!Number.isInteger(Number(articleId))) {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
+
+  return db
+    .query(
+      "SELECT * FROM comments WHERE article_id = $1 ORDER BY created_at DESC",
+      [articleId]
+    )
+    .then((result) => {
+      const comments = result.rows;
```

Comment:
It is not an error if an article has no comments. A new article wouldn't have any.